src/com/linkedlistsample/LinkedListSample.java

```diff
@@ -0,0 +1,39 @@
+package com.linkedlistsample;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+
+public class LinkedListSample {
+	public static void main(String args[]) {
+		LinkedList<Integer> list1 = new LinkedList<>();
+		LinkedList<Integer> list2 = new LinkedList<>();
+		list1.addFirst(5);
```

Comment:
use 'add' instead of 'addFirst'
because we have to create a list in reverse order
e.g if the number is 342 list should be (2 -> 4 -> 3)